src/components/ui/label.tsx

```diff
@@ -0,0 +1,18 @@
+interface InputLabelProps {
+  name: string;
+  required: boolean;
+  html_for: string;
```

Comment:
consider using `camelCase` - `htmlFor` . it is widely used for Javascript ecosystem.